graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DgsDataLoaderProvider.kt

```diff
@@ -203,7 +203,7 @@ class DgsDataLoaderProvider(private val applicationContext: ApplicationContext)
         return DataLoaderFactory.newMappedDataLoader(extendedBatchLoader, options)
     }
 
-    private fun dataLoaderOptions(annotation: DgsDataLoader): DataLoaderOptions {
+    protected open fun dataLoaderOptions(annotation: DgsDataLoader): DataLoaderOptions {
```

Comment:
In your implementation are you overriding only `dataLoaderOptions` or do you also need to override additional methods?

Reply:
Right now only options, however in the future I would need to override how they are obtained to customize them